docs/release-notes/8.3.asciidoc

```diff
@@ -0,0 +1,76 @@
+* Adds a “Recently created cases” widget to the Cases page ({pull}131029[#131029]).
+* Adds a “User risk” tab to the User details flyout ({pull}130256[#130256]).
+* Adds an “Authentications” tab to the User details flyout ({pull}129456[#129456]).
+* Adds the ability to investigate Osquery results in Timeline ({pull}128596[#128596]).
+* Allows multiple alerts to be added to a case ({pull}130958[#130958]).
+* Adds the option to delete case comments multiple alerts to be added to a case ({pull}130254[#130254]).
+* Provides an option to select a severity level for a case ({pull}131626[#131626]).
+* Adds the *Alerts* tab to cases ({pull}131883[#131883]).
+* Adds the *Average time to close* metric to the Cases page ({pull}131909[#131909]).
+
+[discrete]
+[[bug-fixes-8.3.0]]
+==== Bug fixes and enhancements
+* Separates array values in the Alerts table with commas ({pull}133297[#133297]).
+* Exposes the EQL search settings `event_category_field`, `tiebreaker_field`, and `timestamp_field` through the rules API and UI for Event Correlation rules ({pull}132247[#132247]).
+* Adds the *Session ID* field to the *Highlighted fields* section of the Alert details flyout ({pull}132219[#132219]).
+* Introduces a new design for the main navigation menu ({pull}132210[#132210], {pull}131437[#131437]).
+* Adds Dashboards and Threat Hunting Landing pages ({pull}130905[#130905]).
+* Allows highlighted fields to be investigated in Timeline ({pull}131255[#131255]).
+* Adds the *Run Osquery* option to the *More actions* menu (*...*) in the Alerts table ({pull}131790[#131790]).
+* Improves the performance of these actions on the bulk rule actions endpoint ({pull}130924[#130924]).
+** `add_tags`
+** `delete_tags`
+** `set_tags`
+** `add_index_patterns`
+** `delete_index_patterns`
+** `set_index_patterns`
+** `set_timeline`
+* Fixes a bug that caused the rule details page to crash when users opened a deleted or non-existent rule ({pull}133867[#133867]).
+* Allows threshold alerts to be investigated in Timeline if filters are not provided ({pull}133733[#133733]).
+* Prevents events from being added to cases from Timeline ({pull}133410[#133410]).
+* Fixes All Users and All Hosts page should reset the current page when sorting ({pull}133111[#133111]).
+* Removes the filter and investigate in Timeline options from the Elastic Agent status in Highlighted fields ({pull}132829[#132829], {pull}132586[#132586]).
+* Improves the copy of Timeline tooltips ({pull}132756[#132756]).
+* Fixes a validation bug that occurred when users were building a rule exception and changed the exception statement’s operator ({pull}131989[#131989]).
+* Adds a checkmark to the pagination selection on the *Exceptions lists* page ({pull}131979[#131979]).
+* Re-adds the success message that displays when users export an exceptions list ({pull}131952[#131952]).
+* Updates import toast logic ({pull}131873[#131873]).
```

Comment:
Pretty unclear -- maybe a few more words could help:
```suggestion
* Updates notification logic for importing rules ({pull}131873[#131873]).
```

Reply:
Thanks, @joepeeples ! Was saving this till you merged https://github.com/elastic/security-docs/pull/2084.